ds_judgements_public_ui/templates/includes/search/search_help_text.html

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
+<p class="search-term-component__help-text">
+  <a href='{% url "how_to_use_this_service" %}#section-search'>{% translate "basic.search.helptext" %}</a>
```

Comment:
We're generally moving away from translations -- we found them more annoying than helpful; feel free to refactor this to be static test 
https://github.com/nationalarchives/ds-find-caselaw-docs/blob/main/doc/adr/0015-translations.md